docs/source/docker_only.rst

```diff
@@ -0,0 +1,129 @@
+
+If you have these already created you can remove the existing ones and recreate them. To remove the existing ones run:
+
+.. code-block:: shell
+
+    # Needed if you want to recreate containers
+    docker rm dallinger_redis
+    docker rm dallinger_postgres
+
+or you can start the containers you have:
+
+.. code-block:: shell
+
+    # Needed if you want to start stopped containers
+    docker start dallinger_redis dallinger_postgres
+
+Start adminer:
+
+.. code-block:: shell
+
+    docker run -d -p 8080:8080 --name dallinger_adminer --network dallinger --link dallinger_postgres:db adminer
+
+Connect to adminer by visiting http://localhost:8080
+Select PostgreSQL from the dropdown, and enter `dallinger` as both username and password. You can leave the "database" field blank to get a list of existing databases.
+
+
+Run experiment from docker image
+================================
+
+Enter the experiment directory:
+
+.. code-block:: shell
+
+   cd /path/to/experiment-directory
+
+Create a file named `Dockerfile` with these contents (replace image name in the `FROM` directive to match the dallinger version you depend on in your `constraints.txt` file):
+
+.. code-block::
+
+    # syntax = docker/dockerfile:1.2
+    FROM ghcr.io/dallinger/dallinger:9.1.0a1
+
+    RUN mkdir /experiment
+    COPY requirements.txt /experiment/constraints.txt
+
+    WORKDIR /experiment
+
+    RUN python3 -m pip install -e /dallinger[docker]
+
+    # If a dependency needs the ssh client and git, install them
+    RUN grep git+ constraints.txt && \
+        apt-get update && \
+        apt-get install -y openssh-client git && \
+        rm -rf /var/lib/apt/lists || true
+
+    # We rely on the already installed dallinger: the docker image tag has been chosen
+    # based on the contents of this file. This makes sure dallinger stays installed from
+    # /dallinger, and that it doesn't waste space with two copies in two different layers.
+    RUN mkdir ~/.ssh && echo "Host *\n    StrictHostKeyChecking no" >> ~/.ssh/config
+    RUN --mount=type=ssh grep -v ^dallinger constraints.txt > /tmp/requirements_no_dallinger.txt && \
+        python3 -m pip install -r /tmp/requirements_no_dallinger.txt
+    ENV PORT=5000
+    CMD dallinger_heroku_web
+
+Build a docker image for the experiment using Buildkit:
+
+.. code-block:: shell
+    EXPERIMENT_IMAGE=my-experiment
+    DOCKER_BUILDKIT=1 docker build . -t ${EXPERIMENT_IMAGE}
+
+Start the development server with docker:
+
+.. code-block:: shell
+
+    docker run --name dallinger --rm -ti -u $(id -u ${USER}):$(id -g ${USER}) -v ${PWD}:/experiment --network dallinger -p 5000:5000 -e FlASK_OPTIONS='-h 0.0.0.0' -e REDIS_URL=redis://dallinger_redis:6379 -e DATABASE_URL=postgresql://dallinger:dallinger@dallinger_postgres/dallinger ${EXPERIMENT_IMAGE} dallinger develop debug
```

Comment:
Is the lower case "l" by intention (FlASK_OPTIONS)?

Reply:
Indeed! It's fixed now